src/Bootstrap/less/theme/common-readme.less

```
@@ -0,0 +1,12 @@
+.readme{
+    .readme-container{
+        display: block;
+        padding: 10.5px;
+        background-color: #f5f5f5;
+        border:1px solid #ccc;
+        word-break:normal;
```

Nit: please add a space

```suggestion
        word-break: normal;
```

Same comment for line 10